src/v/raft/consensus.cc

```diff
@@ -3350,27 +3350,48 @@ void consensus::maybe_update_majority_replicated_index() {
     _consumable_offset_monitor.notify(last_visible_index());
 }
 
-heartbeats_suppressed consensus::are_heartbeats_suppressed(vnode id) const {
-    if (!_fstats.contains(id)) {
-        return heartbeats_suppressed::yes;
+bool consensus::are_heartbeats_suppressed(vnode id) const {
+    if (auto it = _fstats.find(id); it != _fstats.end()) {
+        it->second.are_heartbeats_suppressed();
```

Comment:
return missing?

Reply:
wow

Reply:
I immediately checked CI and it was green :P

Reply:
Yeah that's surprising. Heartbeats are a conspiracy by Big RPC and not really needed?

Reply:
Turns out this function is not really used (instead `follower_index_metadata::are_heartbeats_suppressed()` is used directly) so I guess I'll just delete it to avoid confusion. Anyway, thanks Bharath Falcon-eyes!

Reply:
phew